_templates/header.php

```diff
@@ -139,6 +139,7 @@
                     <li><a href="https://developer.elementary.io">Developer</a></li>
                     <li><a href="<?php echo $page['lang-root'].'get-involved'; ?>">Get Involved</a></li>
                     <li><a href="<?php echo $page['lang-root'].'store/'; ?>">Store</a></li>
+                    <li><a href="<?php echo 'appstore.'.$page['lang-root']?>">Appstore</a></li>
```

Comment:
This should be `appcenter`, isn't it?

Reply:
the href should be a solid `https://appcenter.elementary.io` and the text should be `AppCenter`.

The whole line should be:
`<li><a href="https://appcenter.elementary.io">AppCenter</a></li>`

I'm checking with design to see if we want this in the header or not.